content/en/blog/_posts/2025-05-22-gateway-api-v1.3/index.md

```diff
@@ -0,0 +1,370 @@
+
+```yaml
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: http-filter-mirror
+  labels:
+    gateway: mirror-gateway
+spec:
+  parentRefs:
+  - name: mirror-gateway
+  hostnames:
+  - mirror.example
+  rules:
+  - backendRefs:
+    - name: foo-v1
+      port: 8080
+    filters:
+    - type: RequestMirror
+      requestMirror:
+        backendRef:
+          name: foo-v2
+          port: 8080
+        percent: 42 # This value must be an integer.
+```
+The percentage can also be expressed using a fraction.  This shows an example
+with 5 out of every 1000 requests to `foo-v1` being mirrored to `foo-v2`.
+
+```yaml
+  rules:
+  - backendRefs:
+    - name: foo-v1
+      port: 8080
+    filters:
+    - type: RequestMirror
+      requestMirror:
+        backendRef:
+          name: foo-v2
+          port: 8080
+        fraction:
+          numerator: 5
+          denominator: 1000
+```
+
+## Additions to Experimental Channel:
+#### Experimental Channel Additions and Changes
+
+The experimental channel is Gateway API's channel for experimenting with new
+features and gaining confidence with them before allowing them to graduate to
+standard.  Please note: the experimental channel may include features that are
+changed or removed later.
+
+Starting in release v1.3.0, in an effort to distinguish experimental channel
+resources from standard channel resources, new experimental resources have the
+prefix "**X**".  For the same reason, experimental resources are now added to the
+object group `gateway.networking.x-k8s.io` instead of `gateway.networking.k8s.io`. 
+Bear in mind that using new experimental channel resources means they can coexist
+with standard channel resources, but migrating these resources to the standard
+channel will require recreating them with the standard channel names and API
+Group (both of which lack the "x" prefix).
+
+This release introduces two new experimental resources: `XBackendTrafficPolicy`
+and `XListenerSet`.
+
+### CORS Filter
+Leads: [Liang Li](https://github.com/liangli), [Eyal Pazz](https://github.com/EyalPazz), [Rob Scott](https://github.com/robscott)
+
+GEP-1767: [CORS Filter](https://github.com/kubernetes-sigs/gateway-api/blob/main/geps/gep-1767/index.md)
+
+Cross-origin resource sharing (CORS) is an HTTP-header based mechanism that allows
+a web page to access restricted resources from a server on an origin (domain,
+scheme, or port) different from the domain that served the web page. This feature
+adds a new `HTTPRouteFilter` called `HTTPCORSFilter` to configure the handling of
+cross-origin requests before the response is sent back to the client.
+
+Here's an example of a simple cross-origin configuration:
+```yaml
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: http-route-cors
+spec:
+  hostnames:
+  - http.route.cors.com
+  parentRefs:
+  - group: gateway.networking.k8s.io
+    kind: Gateway
+    name: http-gateway
+  rules:
+  - backendRefs:
```

Comment:
Very tiny non-blocking nit: Prefer keeping backendRefs to the end and starting with matches so the rule follows "when conditions match then send to backends"